t/t7519-status-show-ignored-directory.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
+#
+#
+
+test_description='git status collapse ignored'
+
+. ./test-lib.sh
+
+
+cat >.gitignore <<\EOF
+*.ign
+ignored_dir/
+!*.unignore
+EOF
+
+test_expect_success 'setup' '
+	mkdir -p tracked tracked_ignored \
+		ignored untracked \
+		untracked_ignored \
+		ignored_dir \
+		dir/ignored_dir \
+		dir/dir/tracked_ignored dir/dir/ignored \
+		dir/dir/untracked dir/dir/untracked_ignored \
+		dir/dir/ignored_dir \
+		dir/dir/ignored_with_sub_untracked \
+		dir/dir/ignored_with_sub_untracked/untracked \
```

Comment:
As `-p` is passed, line 25 is unnecessary, as that directory will be created as part of creating the directory specified in line 26. For the same reason, line 27 is unnecessary.